src/common/Modal/components/ModalContent/index.js

```diff
@@ -1,8 +1,9 @@
-// @ts-check
+import c from "classnames";
```

Comment:
Can you modify the variable's name so it's more descriptive (ex: `classNames`)?

Reply:
Fixed.